process/aws-account-migration.md

```diff
@@ -0,0 +1,223 @@
+
+            billing -.-> reports
+            billing -.- org
+            org -.- policy
+        end
+
+        chickens["🐓 chickens-prod Account<br/><br/>Replay Testing"]
+        devhub["📚 devhub Account<br/><br/>Learn Portal"]
+        docs["📑 docs-prod Account<br/><br/>Docs Portal"]
+        mainnet["⛓️ evm-mainnet Account<br/><br/>EVM Mainnet Endpoints"]
+        testnet["🛠️ evm-testnet Account<br/><br/>EVM Testnet Endpoints"]
+    end
+
+    org ===> chickens
+    org ===> devhub
+    org ===> docs
+    org ===> mainnet
+    org ===> testnet
+```
+Each child account is a "member" of the management account's "organization". The organization policy in the management account enforces consolidated billing, and prevents child accounts from leaving the organization.
+
+Each EVM account looks like this.
+```mermaid
+---
+title: EVM Account Architecture
+---
+flowchart TB
+    subgraph evm["`🛠️ **evm Account**`"]
+        direction TB
+        subgraph policy["`📜 **Policy**`"]
+            direction TB
+            security["🔒 Security"]
+            iam["📋 IAM"]
+            groups["👥 Groups"]
+            users["👤 Users"]
+
+            security -.- iam
+            iam ---> groups
+            users ---> groups
+        end
+
+        subgraph resources["`🏗️ **Resources**`"]
+            direction LR
+            dns["📑 DNS"]
+            cdn["🌐 CDN"]
+            waf["🔥 WAF"]
+            lb["🔀 Load Balancers"]
+            vm["🖥️ VMs"]
+            db["🛢️ Databases"]
+            metrics["📈 Metrics"]
+            notifications["📲 Notifications"]
+
+            dns -.-> cdn
+            cdn <--> waf
+            waf <--> lb
+            lb <--> vm
+            vm <--> db
+
+            dns -.-> metrics
+            cdn -.-> metrics
+            waf -.-> metrics
+            lb -.-> metrics
+            vm -.-> metrics
+
+            metrics ---> notifications
+        end
+
+        groups ---> resources
+    end
+
+    internet((🌐))
+
+    subgraph org["`**ENF Management Account**`"]
+        enf((🏛️))
+    end
+
+    internet -.-> dns
+    internet <==> cdn
+    internet ~~~ evm
+    internet ~~~ iam
+
+    notifications ---> internet
+
+    enf -.-> |DNS| internet
+    resources ---> |💲 Billing| enf
+    enf ===x |Ownership| evm
+```
+You can see each account is isolated with its own security policy, IAM, users, groups, and resources. DNS is delegated using public DNS infrastructure, not Route53 IAM, so DNS will continue to work during and after a migration.
+
+## Prerequisites
+There are a few prerequisites to consider before migrating an AWS account.
+
+### ENF
+The EOS Network Foundation (ENF) will need to do the following.
+1. Determine who will perform the migration, presumably the Automation team.
+1. Grant that user permissions to remove accounts from the organization.
+1. Remove email and SMS subscribers from the notification system in the accounts to be migrated, to protect their personal information.
+
+### EOS Labs
```

Comment:
Can we genericize these instructions such that we do not assume EOS Labs is the organization on the receiving end.  I want to avoid future confusion if there is a different 3rd party and EOS Labs has no part in the process.

Reply:
I have renamed all visible references. I can rename invisible references as well, if needed.